src/api/useFetch.js

```diff
@@ -142,6 +156,7 @@ const _useFetch = ({
           if (!retryIntervalRef.current) {
             retryIntervalRef.current = setInterval(() => fetchData(), retryIntervalSeconds * 1000)
           }
+          isFetching.current = false
```

Comment:
This can be dropped since it's part of the `finally` clause anyways.

Reply:
I do not think so since the `return` statement is reached and immediately exists the function (in that: skipping the finally block).

Reply:
I withdraw my statement, you are right -> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch#the_finally_block